lib/docker/api/exec.ex

```diff
@@ -29,6 +29,14 @@ defmodule Docker.Api.Exec do
     end
   end
 
+  def stdout_logs(container_id) do
+    case Client.get("/containers/#{container_id}/logs?stdout=true") do
```

Comment:
Should look into streaming. The logs can get very large. And its basically getting the whole stdout from the container.